spec/boolean_spec.rb

```diff
@@ -256,14 +256,14 @@
     it 'lesbian OR gay videos', jira: ['VUF-300', 'VUF-301', 'VUF-311'] do
       resp = solr_resp_doc_ids_only('q' => 'lesbian OR gay', 'fq' => 'format:Video')
       expect(resp.size).to be >= 1000
-      expect(resp.size).to be <= 1750
+      expect(resp.size).to be <= 1770
```

Comment:
for the future, in general:   I suspect the max here has drifted up a lot since this test was originally written ... and the min was never raised.  A 770 record diff between min and max seems large.  maybe ... 250?  would be an appropriate target diff in general?